examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3067,6 +3067,28 @@ server cluster ColorControl = 768 {
   command StepColorTemperature(StepColorTemperatureRequest): DefaultSuccess = 76;
 }
 
+server cluster BallastConfiguration = 769 {
+  readonly attribute int8u physicalMinLevel = 0;
+  readonly attribute int8u physicalMaxLevel = 1;
+  readonly attribute bitmap8 ballastStatus = 2;
+  attribute int8u minLevel = 16;
+  attribute int8u maxLevel = 17;
+  attribute nullable int8u intrinsicBalanceFactor = 20;
+  attribute nullable int8u ballastFactorAdjustment = 21;
+  readonly attribute int8u lampQuantity = 32;
+  attribute char_string<16> lampType = 48;
```

Comment:
How did we decide which of the optional vs mandatory attributes we support?

All these lap settings attributes are optional and are here, however things like intrinsicBallastFactor are optional and are not here.

Reply:
I changed it according to @bzbarsky-apple comment above. Everything on in all-clusters-app, only mandatory attributes on in all-clusters-minimal-app.

Attributes PowerOnLevel and PowerOnFadeTime are deprecated.

The ha.xml had this wrongly named Intrinsic**Balance**Factor. Fixed.